streams-tests/shared/src/test/scala/zio/stream/ZStreamSpec.scala

```diff
@@ -203,6 +203,19 @@ object ZStreamSpec extends ZIOBaseSpec {
               equalTo(Chunk(Right(List(2, 1, 1, 1, 1)), Right(List(2))))
             )
           },
+          testM("fails fast") {
+            for {
+              queue <- Queue.unbounded[Int]
+              _ <- ZStream
+                     .range(1, 10)
+                     .tap(i => ZIO.fail("BOOM!").when(i == 6) *> queue.offer(i))
+                     .aggregateAsyncWithin(ZTransducer.foldUntil((), 5)((_, _) => ()), Schedule.fixed(5.seconds))
```

Comment:
This could very well use `Schedule.forever` as we're not advancing the test clock anywhere, right?

Reply:
True that, changed.